moped-editor/src/utils/dateAndTime.js

```diff
@@ -57,11 +57,58 @@ export const makeHourAndMinutesFromTimeStampTZ = timeStampTZString => {
  * @param {String} timeStampTZString - Timestamptz string in ISO format
  * @return {String} Time formatted as HH:MM:SS XM
  */
-export const makeFullTimeFromTimeStampTZ = timeStampTZString => {
+export const makeFullTimeFromTimeStampTZ = (timeStampTZString) => {
   const dateObject = new Date(timeStampTZString);
   return dateObject.toLocaleTimeString("en-US", {
     hour: "numeric",
     minute: "numeric",
     second: "numeric",
   });
 };
+
+const MILLS_CONVERSION = {
+  minute: 60_000,
+  hour: 3_600_000,
+  day: 86_400_000,
+};
+
+/**
+ * Format millseconds into a relative description
+ * @param {integer} millsElapsed - the elapsed time in milliseconds to format
+ * @param {string} unitName - the name of the units to convert to - the string must
+ * be defined in the MILLS_CONVERSION objeect
+ * @returns {string} in format  `<value> <unit-name> ago`
+ */
+const formatRelativeTime = (millsElapsed, unitName) => {
```

Comment:
i had fun with this coding challenge, but i am very open to using a library instead. [this one](https://github.com/EvanHahn/HumanizeDuration.js) works well.

Reply:
I think this is great without the library - especially with test coverage! 🙌

Reply:
I agree wholeheartedly with Mike - this works great. A dependency we didn't we didn't pick up is 👍. 🚀